src/request/export/index.html

```diff
@@ -268,6 +277,44 @@ <h1 data-i18n="export-file-heading" class="nq-h1">Download Login File</h1>
                 </div>
             </div>
 
+            <div id="login-file-success" class="page nq-card">
+                <div class="page-body nq-card-body">
+                    <button id="close-request">
+                        <svg class="nq-icon"><use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-close"/></svg>
+                    </button>
+                    <ul>
+                        <li class="nq-green">
+                            <svg class="nq-icon">
+                                <use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-checkmark"/>
+                            </svg>
+                            <span data-i18n="export-file-success-set-password">Set password</span>
+                        </li>
+                        <li class="nq-green">
+                            <svg class="nq-icon">
+                                <use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-checkmark"/>
+                            </svg>
+                            <span data-i18n="export-file-success-save-file">Save Login File</span>
+                        </li>
+                        <li class="nq-grey">
+                            <span class="nq-icon bullet-point"></span>
+                            <span data-i18n="export-file-success-create-backup">Create backup</span>
+                        </li>
+                    </ul>
+                    <h1 class="nq-h1" data-i18n="export-file-success-heading">Take 5 minutes for a backup</h1>
+                    <p class="nq-text" data-i18n="export-file-success-words-intro">There is no ‘forgot password’ option. Write down 24 words to create a secure backup.</p>
```

Comment:
I don't know which apostrophe is correct.

```suggestion
                    <p class="nq-text" data-i18n="export-file-success-words-intro">There is no 'forgot password' option. Write down 24 words to create a secure backup.</p>
```